test/models/repository_test.rb

```diff
@@ -881,6 +882,20 @@ def test_search_content_view_id
       assert_includes repos, @fedora_17_x86_64
     end
 
+    def test_search_description
+      @repo = build(:katello_repository, :fedora_17_el6,
```

Comment:
You'll notice that other tests in this class are using repos created via fixtures to make their assertions rather than what you've done here - building an entirely new repo. There's nothing inherently wrong with your approach but I would suggest modifying an existing repo fixture - such as `fedora_17_x86_64` to give it a description and test against that fixture. Why?

- less happening in this test in terms of LOC/complexity
- putting a desc on the repo I mentioned subjects the existence of that field to more exposure since this fixture is used all over the place
- fewer queries into the db (not creating this repo) and since our test suite is long-running it's better to save time whenever possible